apps/web/src/pages/TalentNominations/NominationGroup/Profile.tsx

```diff
@@ -1,40 +1,212 @@
+            Icon={UserCircleIcon}
+            color="primary"
+            data-h2-margin="base(0)"
+          >
+            {intl.formatMessage({
+              defaultMessage: "Profile and career plan",
+              id: "mSj3fM",
+              description:
+                "Heading for nominee profile page accordion sections",
+            })}
+          </Heading>
+          <div
+            data-h2-display="base(flex)"
+            data-h2-align-items="base(flex-end)"
+            data-h2-gap="base(0 x.5)"
+          >
+            <Button mode="inline" color="secondary" onClick={toggleSections}>
+              {hasOpenSections
+                ? intl.formatMessage({
+                    defaultMessage:
+                      "Collapse all <hidden>profile and career plan</hidden>sections",
+                    id: "/UoSSQ",
+                    description:
+                      "Button text to close all profile and career plan accordions",
+                  })
+                : intl.formatMessage({
+                    defaultMessage:
+                      "Expand all <hidden>profile and career plan</hidden>sections",
+                    id: "tS5VSg",
+                    description:
+                      "Button text to open all profile and career plan accordions",
+                  })}
+            </Button>
+          </div>
+        </div>
+        <p data-h2-margin="base(x1 0)">
+          {intl.formatMessage({
+            defaultMessage:
+              "The following sections can be expanded to show information about the nominee’s profile, their interest in this community, and their career goals.",
+            id: "xA7XvK",
+            description:
+              "Description for the nominee profile page accordion sections",
+          })}
+        </p>
+        {!shareProfile && (
+          <Well color="error">
+            <p data-h2-margin-bottom="base(x1)">
+              {intl.formatMessage({
+                defaultMessage:
+                  "This nominee has not agreed to share their information with your community",
+                id: "4ujr5X",
+                description: "Null message for nominee profile",
+              })}
+            </p>
+            <p>
+              {intl.formatMessage({
+                defaultMessage:
+                  "Nominees can agree to provide access to their profile using the “Functional communities” tool on their dashboard.",
+                id: "8plD42",
+                description: "Null secondary message for nominee profile",
+              })}
+            </p>
+          </Well>
+        )}
+      </CardBasic>
+      {data?.user?.employeeProfile ? (
+        <Accordion.Root
+          type="multiple"
+          mode="card"
+          size="sm"
+          value={openSections}
+          onValueChange={setOpenSections}
+          data-h2-margin-top="base(0)"
+          data-h2-border-radius="base(0 0 6px 6px)" // TODO: Won't it override accordion radius
```

Comment:
I think you can do "simple" accordion mode inside the above card to remove that weird issue.

Reply:
Fixed in [Fix heading section radius and background color styling](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13300/commits/780c6b358e5f935de325769304d4a4e42f55a963).

Reply:
I'd prefer if we could avoid in-line styles and use `h2` but I guess it technically works so :woman_shrugging: